test/test_nestedtensor.py

```diff
@@ -0,0 +1,146 @@
+import torch
+import unittest
+from common_utils import TEST_WITH_PROTOTYPES
+
+if TEST_WITH_PROTOTYPES:
+    import torch.prototypes.nestedtensor as nestedtensor
+    from common_utils import TestCase
+    NestedTensor = nestedtensor.NestedTensor
+else:
+    print('prototypes not available, skipping tests')
```

Comment:
There's a way to do this using unittest.skipIf

Reply:
I copied this from test_cuda - will look into it

Reply:
It's probably better the way you have it, the alternative is to apply unittest.skipif to every test in this file.